src/app/workflow/dag/wdl-viewer/state/wdl-viewer.model.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ * Defines types for the response of Pipeline Builder library
+ */
+export interface WdlViewerPipelineResponse {
+  status: boolean;
+  message: string;
+  model: Array<any>;
+  actions: Array<any>;
+
+  // Not originally part of the response object, but all responses should be identified by the workflow version used to generated it
+  workflowVersion?: number;
```

Comment:
Looking at the code as it is now, this feels weird to me. The only place you use this is in `wdl-viewer-service.update()`. I would suggest getting rid of this from here, and the interface will cleanly model the response you get from the library, and then just adding the workflow version id as an additional parameter to the update method.